jobs/healing/pipeline-healing.js

```diff
@@ -229,23 +232,91 @@ async function getScopedSourceTriples( config, property, conceptSchemeUri, publi
   return reformatQueryResult(result);
 }
 
+function arrayToFile(array, file){
+  let fd = fs.openSync(file, "w");
+  for (let i=0; i<array.length; ++i){
+    fs.writeSync(fd, JSON.stringify(array[i]) + "\n")
+  }
+  fs.close(fd)
+  return file
+}
+function getTempFile() {
```

Comment:
perhaps considering a library? (not sure but there is some cleaning boilerplate etc that might be easier)

Reply:
do you have any libraries in mind to do this?

Reply:
lib tmp? (but only if's worth the boilerplate)